plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1909,6 +1911,209 @@ read_only::get_table_rows_result read_only::get_table_rows( const read_only::get
 #pragma GCC diagnostic pop
 }
 
```

Comment:
Are the `chain_api_plugin` changes coming in under a diff PR?

Reply:
It is not in the original scope.  I can add one line but we need to have a separate task to test http plugin.

Reply:
This is unusable without http support. If there is a JIRA issue to add that, then fine otherwise please create a JIRA issue to add http support.

Reply:
added chain_api_plugin change.

Reply:
Make sure to update the PR description with docs on the new apis.

Reply:
Also add a test for it to `plugin_http_api_test.py`. Also what about `cleos` support? Is that a diff PR?

Reply:
Cleos support is a separate task by a different person

Reply:
added get_kv_table_rows to plugin_http_api_test.

Reply:
updated chain.swagger.yaml.